build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld

```diff
@@ -253,8 +253,10 @@ SECTIONS
 		__preinit_array_end = .;
 		. = ALIGN(4);
 		__init_array_start = .;
+		_sinit = ABSOLUTE(.);
```

Comment:
Could you split commits for this ld change?
1. configs/rtl8721csm/scripts: add _sinit, _einit to support c++
2. configs/rtl8721csm: add cxxtest config

Reply:
I made new PR
https://github.com/Samsung/TizenRT/pull/5383